plugins/google_directory/icon_google_directory/util/tools.py

```diff
@@ -22,3 +23,25 @@ def return_contact_informations(input_dict: dict) -> dict:
         if "postalCode" in address:
             address["postal_code"] = address.pop("postalCode")
     return {"addresses": addresses_list, "phone_numbers": phones_list, "email_addresses": emails_list}
+
+
+def return_contact_informations_name(input_dict: dict) -> dict:
+    """return_contact_informations. Function allows to map data that contains contact informations retrieved from
+    Google API users().list()
+
+    :param input_dict: Input dict as a response got from Google API
+    :type input_dict: dict
+
+    :returns: Mapped dictionary that contains keys such as 'addresses', 'phones', and 'emails'
+    :rtype: dict
+    """
+    for user in input_dict.get("users"):
+        addresses_list = user.get("addresses", [])
+        phones_list = [element.get("value") for element in user.get("phones", [])]
+        emails_list = [element.get("address") for element in user.get("emails", [])]
+        for address in addresses_list:
+            if "streetAddress" in address:
+                address["street"] = address.pop("streetAddress")
+            if "postalCode" in address:
+                address["postal_code"] = address.pop("postalCode")
+        return {"addresses": addresses_list, "phone_numbers": phones_list, "email_addresses": emails_list}
```

Comment:
I am just wondering if it's possible to use above function ``return_contact_informations` in a for loop, and append lists from the data there. That would save a few lines of code, and allows avoiding doubling the code 😃